Extension/src/main.ts

```diff
@@ -488,6 +488,7 @@ function rewriteManifest(): Promise<void> {
         "onCommand:C_Cpp.GenerateEditorConfig",
         "onCommand:C_Cpp.GoToNextDirectiveInGroup",
         "onCommand:C_Cpp.GoToPrevDirectiveInGroup",
+        "onCommand:C_Cpp.CheckForCompiler",
```

Comment:
This should be added to commandsToRegister in commands.ts.

Reply:
Actually, it looks like it doesn't belong in commandsToRegister since it isn't callable via the command palette.